.github/workflows/ci-test-xtgeo-cibuildwheel.yml

```diff
@@ -0,0 +1,67 @@
+# Test building some cibuildwheel on pull requests (simplified testing)
+
+name: cibuildwheel
+
+on:
+  pull_request:
+    branches: [master]
+
+env:
+  CIBW_BEFORE_ALL_LINUX: "sh scripts/swig_manylinux.sh"
+  CIBW_BEFORE_ALL_WINDOWS: "choco install -y --no-progress --allow-empty-checksums -r swig --version=4.0.1"
+  CIBW_BEFORE_ALL_MACOS: brew install swig
+
+  CIBW_BEFORE_TEST: |
+    pushd {project}
+    pip install -r requirements/requirements.txt
+    pip install -r requirements/requirements_test.txt
+
+  # 3.9 have trouble with install pytables on Mac and Win, due to missing wheel
+  # so full testing on those platforms are postponed.
+  CIBW_TEST_COMMAND_MACOS: |
+    pushd {project}
+    rm -rf ../xtgeo-testdata
+    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+    pytest tests/test_surface --disable-warnings
+  CIBW_TEST_COMMAND_WINDOWS: |
+    pushd {project}
+    rmdir /s ../xtgeo-testdata
+    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+    pytest tests/test_surface --disable-warnings
```

Comment:
Doesn't look like this part is run completely. For both `osx` and `ubuntu` I see the tests are run (subset of them on `osx` of course).

For `win` it doesn't even look like it's downloading the `xtgeo-data` repo

Reply:
Yes mananged finally to detect the cause: In windows, only the first line was executed(!), as the YAML line break is "\n" which is not parsed properly in windows. So the solution is to replace `|` with > and use` &&`:
```
CIWB_TEST_COMMAND: > 
   command1 &&
   command2
```

Reply:
That is indeed an annoying pitfall.. Could even warrant a comment to make sure later developers won't swap back in case of preferences..

Reply:
Good idea; a comment is given